src/masternodes/rpc_evm.cpp

```diff
@@ -122,11 +139,186 @@ UniValue evmtx(const JSONRPCRequest& request) {
     return send(MakeTransactionRef(std::move(rawTx)), optAuthTx)->GetHash().ToString();
 }
 
+UniValue evmrawtx(const JSONRPCRequest& request) {
```

Comment:
Remove evmrawtx. I think this was added in by accident as it is not used in this PR and is not added to commands below so is not available. We have removed this RPC call as raw TXs can now be sent direct against EVM in functional tests.